pom.xml

```diff
@@ -109,6 +109,16 @@
             <artifactId>sharepoint-services-java</artifactId>
             <version>1.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.2.3</version>
+        </dependency>
```

Comment:
You only need `logback-classic` here BTW

Reply:
cool I have removed it